packages/react-atlas-default-theme/webpack.config.js

```diff
@@ -16,6 +16,7 @@ module.exports = {
     rules: [
     {
       test: /\.css$/,
+      exclude: /\.global\.css$/,

```

Comment:
Why are you excluding the global CSS, shouldn't we let the css loader load the CSS?